Docs/LinuxOsxInstall.md

```diff
@@ -13,15 +13,21 @@
 - Make sure you have OpenGL availible
 
 ## Linux installation instructions
+### Arch 
+The AUR package `factorio-yafc-git` points to the original upstream project by ShadowTheAge. Download the PKGBUILD and edit the url to the following: https://github.com/have-fun-was-taken/yafc-ce
```

Comment:
```suggestion
The AUR package `factorio-yafc-git` points to the original upstream project by ShadowTheAge. Clone the AUR package and edit the url to the following: https://github.com/have-fun-was-taken/yafc-ce
```

The package needs the desktop file as well, so only fetching the PKGBUILD would not work

Reply:
Thanks for the catch - Im actually using my own PKGBUILD and didn't notice that their desktop file is uploaded to the AUR.

Reply:
I was curious about the AUR and checked it out myself :)

Reply:
If the packager is not willing to package YAFC-CE as well, I can make one if needed. But let's wait their response first